config/registry/aws/modules/aws-dns.md

```diff
@@ -11,3 +11,73 @@ This module creates a [Route53 hosted zone](https://docs.aws.amazon.com/Route53/
 your given domain. The [k8s-base]({{< relref "#k8s-base" >}}) module automatically hooks up the load balancer to it
 for the domain and subdomain specified, but in order for this to actually receive traffic you will need to complete
 the [dns setup](/tutorials/ingress).
+
+
+## Changing dns domains
+Currently, Opta does not support multiple domains (except subdomains) per environment, although that may change
+based on user demand. In order for a user to change the domain tied to their environment, they would need to
+remove their current dns module, and then add a new one after applying.
+
+Take for example, if we had the current running environment:
+
+```yaml
+name: blah
```

Comment:
nit: "blah" and "baloney" are kind of negative words. Let's put something more meaningful here which reinforces other stuff for us. Like name: "staging", org_name: "runx" and then domain: "example.com" and "new-domain.com"